_posts/2023-12-14-cockpit-307.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 307
+author: jelle
+date: '2023-12-14'
+tags: cockpit, machines
+slug: cockpit-307
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 307 and cockpit-machines 304:
+
+
+## Storage redesign
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/19472>
+{:.note}
+
+The Storage page has been redesigned with the goal of making more things visible more of the time. The overview page now shows all storage objects in a table and allows almost all operations to be performed on them right from the table.
+
+Clicking on a row navigates to a page dedicated to the object in that row, with more information and maybe a few more actions.
+
+We will continue to improve the Storage pages, by adding filtering options to the big table and working on the actions and dialogs.
+
+![screenshot of ## storage redesign]({{ site.baseurl }}/images/307-storage-redesign.png)
```

Comment:
This should be updated to 

![image](https://github.com/cockpit-project/cockpit-project.github.io/assets/3228183/ab1877c8-3c4e-415c-9cb1-06411db91570)